.github/workflows/ci-pr.yml

```diff
@@ -101,3 +101,44 @@ jobs:
           --enable-scheme=${{ github.workspace }}/host-racket/src/build/cs/c
         make
         make install
+
+  build-pb-ios:
+    runs-on: macos-14
+    steps:
+    - uses: actions/checkout@v4
+    - name: Fetch LibFFI
+      run: |
+        set -euxo pipefail
+        curl -O https://racket.defn.io/libffi-aarch64-ios-3.4.6.tar.gz
+        echo '1a312af2ef169f40bbc9a2e9bdbd52b9e8de35e7  libffi-aarch64-ios-3.4.6.tar.gz' | shasum --check -
```

Comment:
I'm a bit anxious about downloading built artifacts from non-Racket infra like this, but I haven't been able to get LibFFI to cross-compile for iOS inside GH Actions. Other options here would be:

1. Defer adding this build job until I can figure out how to get LibFFI to compile in GHA.
2. Host the built archive somewhere in the Racket infra (maybe even a GH repo?).
3. Keep as-is since the checksum protects us from tampering.